### ISSUE TITLE
[Bug] "The media could not be loaded, either because the server or network failed or because the format is not supported."

### ISSUE THREAD

**Issue by DUOLabs333** [iv-org/inv_sig_helper#25]
**Describe the bug**
For _almost_ all videos (shorts and a few other videos seem to be immune), I started getting this error: "The media could not be loaded, either because the server or network failed or because the format is not supported." This started about 2 days ago.

**Steps to Reproduce**
1. Go to https://<invidious instance>/watch?v=tPTvG0Se7L8
2. See error

**Logs**

**Screenshots**

**Additional context**
 - Browser (if applicable): Chrome 120
 - OS (if applicable): macOS 11

I tried adding the `po_token` and `visitor_data` from https://github.com/iv-org/youtube-trusted-session-generator, but it didn't help.

Switching to `master-arm64` also didn't help.

**Comment by unixfox** [comment]
Ping @techmetx11

**Comment by TropicZL1** [comment]
Was having the same issue today. Stumbled across this issue but was able to fix it by amending my compose yaml. Had to add inv_sig_helper from the https://docs.invidious.io/installation/ (docker-compose method (production)) and then subsequently add visitor data and po token. @DUOLabs333 Did you also do this or did you only add the token and data?

**Comment by DUOLabs333** [comment]
I didn't set up the sig_helper --- I didn't even know that existed until today.

**Comment by TropicZL1** [comment]
I suspect that your issue then is probably that you set up your token and data but are not running the sig helper. Add `signature_server: inv_sig_helper:12999` to your yaml under Invidious and then add

`inv_sig_helper:
    image: quay.io/invidious/inv-sig-helper:latest
    command: ["--tcp", "0.0.0.0:12999"]
    environment:
      - RUST_LOG=info
    restart: unless-stopped
    cap_drop:
      - ALL
    read_only: true
    security_opt:
      - no-new-privileges:true`
      
    Then simply recreate and try again.

**Comment by DUOLabs333** [comment]
I set up everything, and I get the same error.

**Comment by DUOLabs333** [comment]
Oh, I think it does work, but cached videos (even those that were opened unsuccessfully) are not sent to the helper.

**Comment by DUOLabs333** [comment]
@unixfox  How do you drop cached videos from the database? I used to have it in my container-compose.py, but I must have deleted it (probably because I had no need for it).

**Comment by TropicZL1** [comment]
Was just about to comment with my compose.yaml. Mine seems to be opening even videos that didn't successfully open before. Did you fully bring down the containers using `docker compose down` and bringing it back up with `docker compose up -d`?

**Comment by DUOLabs333** [comment]
Yes --- after `stop`, I checked that no Invidious-related processes are running and that the container is no longer mounted.

**Comment by TropicZL1** [comment]
Are you managing your containers in Portainer?

**Comment by DUOLabs333** [comment]
> Oh, I think it does work, but cached videos (even those that were opened unsuccessfully) are not sent to the helper.

Yeah, truncation was needed (I remembered the command --- `psql -d invidious -c 'TRUNCATE TABLE videos`)

> Are you managing your containers in Portainer?

No, I use my own scripts.

**Comment by TropicZL1** [comment]
Gotcha. So it seems sig helper must be necessary now for normal function of Invidious. Since I first posted about an hour ago I haven't run into any issues so please keep us all updated if you run into any issues yourself.

**Comment by DUOLabs333** [comment]
Yeah, disabling the helper brings back the error.

**Comment by DUOLabs333** [comment]
Should this be closed now, with maybe an update to invidious' README?

**Comment by danvolchek** [comment]
Also fixed the problem for me; also did not know that this was needed. It's in the [install docs](https://docs.invidious.io/installation/#docker-compose-method-production) already, but I didn't know it changed/had an update.

Some sort of notification for new install requirements would be cool!

Thanks for the info/fix :)
